src/builtin/modules/math.rs

```diff
@@ -102,7 +102,74 @@ fn clamp(mut args: CallArgs, parser: &mut Parser<'_>) -> SassResult<Value> {
 }
 
 fn hypot(mut args: CallArgs, parser: &mut Parser<'_>) -> SassResult<Value> {
-    todo!()
+    args.min_args(1)?;
+
+    let span = args.span();
+
+    let mut numbers = args.get_variadic()?.into_iter().map(|v| -> SassResult<_> {
+        match v.node {
+            Value::Dimension(n, u, ..) => Ok((n, u)),
+            v => Err((format!("{} is not a number.", v.inspect(span)?), span).into()),
+        }
+    });
+
+    let first: (Number, Unit) = match numbers.next().unwrap()? {
```

Comment:
I think maybe we should name this `(firstn, firstu)` rather than using `first.0` later on?